supp2u/src/components/customerviews/CustomerCart.sass

```diff
@@ -62,7 +62,7 @@
     display: inline-block;
     font-size: 16px;
 
-.check-info button
+.checkoutbox button
     background-color: #bb1535
     border: none;
```

Comment:
no need for ';' in sass

Reply:
first time in sass... but yeah, they don't seem needed, just css/less muscle memory kicking in ;p